### PR TITLE
[PHP] Introduce swoft

### PR DESCRIPTION
**Pull request by waghanza**
Hi,

This `PR` adds [swoft](https://swoft.org)

Closes https://github.com/the-benchmarker/web-frameworks/issues/1305 and https://github.com/the-benchmarker/web-frameworks/issues/1502

/cc @stelin @huangzhhui @inhere could you review ?

### CODE REVIEW

**Comment by waghanza** [comment]
@huangzhhui @inhere could you clean / remove unused files ? (I am only discovering `swoft`)
I prefer having very `thin` implementations here

**Comment by inhere** [comment]
Oh, OK. I am very happy to be able to help you.

**Comment by waghanza** [comment]
@inhere The simplest solution for me is that you fork my branch

**Comment by inhere** [comment]
@waghanza  ok

**Comment by inhere** [comment]
hi @waghanza  I have a PR push to your repository

**Comment by inhere** [comment]
hi @waghanza  I have a PR push to your repository

File: php/swoft/app/Application.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * @since 2.0
+ */
+class Application extends SwoftApplication
+{
+    protected function beforeInit(): void
+    {
+        parent::beforeInit();
+
+        date_default_timezone_set('Asia/Shanghai');
```
Comment (waghanza) [inline php/swoft/app/Application.php]: really needed ?

File: php/swoft/app/Helper/Functions.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * Custom global functions
+ */
+
+function user_func(): string
```
Comment (waghanza) [inline php/swoft/app/Helper/Functions.php]: when this function is called ?

File: php/swoft/app/Application.php (hunk 1 above)
Comment (inhere) [reply php/swoft/app/Application.php]: not, you can remove it.

File: php/swoft/app/Helper/Functions.php (hunk 2 above)
Comment (inhere) [reply php/swoft/app/Helper/Functions.php]: unused, can remove

File: php/swoft/app/Exception/ApiException.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (inhere) [inline php/swoft/app/Exception/ApiException.php]: the file can remove

File: php/swoft/app/Exception/Handler/ApiExceptionHandler.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Exception/Handler/ApiExceptionHandler.php]: unused

File: php/swoft/app/Http/Controller/CoController.php [hunk 5]
```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Http/Controller/CoController.php]: unused file

File: php/swoft/app/Http/Controller/ExceptionController.php [hunk 6]
```diff
@@ -0,0 +1,54 @@
+<?php
```
Comment (inhere) [inline php/swoft/app/Http/Controller/ExceptionController.php]: unused file

File: php/swoft/app/Http/Controller/LogController.php [hunk 7]
```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Http/Controller/LogController.php]: unused file

File: php/swoft/app/Http/Controller/RespController.php [hunk 8]
```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Http/Controller/RespController.php]: unused file

File: php/swoft/app/Model/Dao/UserDao.php [hunk 9]
```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Model/Dao/UserDao.php]: unused file

File: php/swoft/app/Model/Data/UserData.php [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Model/Data/UserData.php]: unused file

File: php/swoft/app/Model/Logic/RequestBean.php [hunk 11]
```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
```
Comment (inhere) [inline php/swoft/app/Model/Logic/RequestBean.php]: unused file

File: php/swoft/app/Model/Logic/UserLogic.php [hunk 12]
```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
```
Comment (inhere) [inline php/swoft/app/Model/Logic/UserLogic.php]: unused file

File: php/swoft/config/dev/db.php [hunk 13]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (inhere) [inline php/swoft/config/dev/db.php]: unused file

**Comment by waghanza** [comment]
@inhere @huangzhhui  `swoft` has been added to this list (and results are available on `README`)

**Pull request merged**